uber/decorators.py

```diff
@@ -270,7 +270,7 @@ def renderable_data(data=None):
 def render(template_name_list, data=None):
     data = renderable_data(data)
     env = JinjaEnv.env()
-    template = env.get_template(template_name_list)
+    template = env.get_or_select_template(template_name_list)
```

Comment:
This line is not related to this fix, but I ran across it trying to track down this bug. `get_or_select_template` is the correct method call.